test/acceptance/tools/rules_utils.py

```diff
@@ -79,7 +79,78 @@
+SC_REGEXP            = u'regexp'
+SC_TYPE              = u'type'
+SC_VALUE_THRESHOLD   = u'valueThreshold'
+SC_ATTR_THRESHOLD    = u'attributeThreshold'
+SC_CEP_EPL           = u'ceprule'
+AC_SMS_TYPE          = u'SendSmsMibAction'
+AC_EMAIL_TYPE        = u'SendEmailAction'
+AC_UPDATE_TYPE       = u'updateAttribute'
+TC_TIME_ELAPSED      = u'timeElapsed'
+
+VISUAL_RULES         = u'visual_rules'
+CONNECTED_TO         = u'connectedTo'
+CONDITION_LIST       = u'conditionList'
+SENSOR_DATA          = u'sensorData'
+CONFIG_DATA          = u'configData'
+TIME_DATA            = u'timeData'
+TIME_TYPE            = u'timeType'
+PARAMETER_VALUE      = u'parameterValue'
+OPERATOR             = u'operator'
+OP_EQUAL_TO          = u'EQUAL_TO'
+OP_MATCH             = u'MATCH'
+OP_DIFFERENT_TO      = u'DIFFERENT_TO'
+OP_GREATER_THAN      = u'GREATER_THAN'
+INTERVAL             = u'interval'
+MEASURE_NAME         = u'measureName'
+DATA_TYPE            = u'dataType'
+DATA_TYPE_TEXT       = u'Text'
+SCOPE                = u'scope'
+SCOPE_LAST_MEASURE   = u'LAST_MEASURE'
+SCOPE_XPATH          = u'XPATH'
+SCOPE_OBSERVATION    = u'OBSERVATION'
+NOT                  = u'Not'
+REPEAT               = u'repeat'
+REPEAT_VALUE         = u'-1'
+CONTEXT              = u'context'
+ASSET                = u'ASSET'
+PARAMETER_NAME       = u'parameterName'
+EPL                  = u'EPL'
+
+AC_MAIL_FROM         = u'mail.from'
+AC_MAIL_FROM_VALUE   = u'iot_support@tid.es'
+AC_MAIL_TO           = u'mail.to'
+AC_MAIL_SUBJECT      = u'mail.subject'
+AC_MAIL_MESSAGE      = u'mail.message'
+AC_MAIL_SUBJ_VALUE   = u'IoT message'
+AC_SMS_NAME          = u'sms'
+AC_SMS_TO            = u'sms.to'
+AC_SMS_MESSAGE       = u'sms.message'
+RESPONSE             = u'response'
+ACTION_DATA          = u'actionData'
+USER_PARAMS          = u'userParams'
+ACTIVE               = u'active'
+CARDS                = u'cards'
+X_AUTH_TOKEN         = u'X-Auth-Token'
+X_AUTH_TOKEN_VALUE   = u'MCAQExCTAHBgUrDgMCGjCCAlwGCSqGSIb3DQEHAaCCAk0EggJJeyJ0b2tlbiI6IHsibWV0aG9kcyI6IFsidG9rZW4iLCAicGFzc3dvcmQiXSwgInJvbGVzIjogW3siaWQiOiAiMjkxOWYzZTRjYTJlNGI3Mjg0OWJmZDAxZGIwZThiMWYiLCAibmFtZSI6ICJkNGNkNjcwOTNmMWQ0Y2U4OWNiODUwYWU3MWQ4NDgxNyNTdWJTZXJ2aWNlQWRtaW4ifV0sICJleHBpcmVzX2F0IjogIjIwMTUtMDItMDRUMTA6Mzg6NDcuODEyOTk1WiIsICJwcm9qZWN0IjogeyJkb21haW4iOiB7ImlkIjogImQ0Y2Q2NzA5M2YxZDRjZTg5Y2I4NTBhZTcxZDg0ODE3IiwgIm5hbWUiOiAiU21hcnRWYWxlbmNpYSJ9LCAiaWQiOiAiMjVkM2ViZDljMWIxNDE3ZDk5NGI5MGYzN2NmM2YxYmUiLCAibmFtZSI6ICIvRWxlY3RyaWNpZGFkIn0sICJjYXRhbG9nIjogW10sICJleHRyYXMiOiB7fSwgInVzZXIiOiB7ImRvbWFpbiI6IHsiaWQiOiAiZDRjZDY3MDkzZjFkNGNlODljYjg1MGFlNzFkODQ4MTciLCAibmFtZSI6ICJTbWFydFZhbGVuY2lhIn0sICJpZCI6ICJkOWQ1NGVjMjE1NjM0OWI4OTRlNTVmMjE2NWM1MmQ4MyIsICJuYW1lIjogIkFsaWNlIn0sICJpc3N1ZWRfYXQiOiAiMjAxNS0wMi0wNFQwOTozODo0OS4zMzM2MDBaIn19MYIBgTCCAX0CAQEwXDBXMQswCQYDVQQGEwJVUzEOMAwGA1UECAwFVW5zZXQxDjAMBgNVBAcMBVVuc2V0MQ4wDAYDVQQKDAVVbnNldDEYMBYGA1UEAwwPd3d3LmV4YW1wbGUuY29tAgEBMAcGBSsOAwIaMA0GCSqGSIb3DQEBAQUABIIBAJmYKdlwuXALebkqt'
```

Comment:
That´s right, this code is not valid, but to try to simulate a request from portal, is necessary a value. Changed to fake value.